proto/viam/service/slam/v1/slam.proto

```diff
@@ -123,6 +145,30 @@ message GetInternalStateRequest {
 }
 
 message GetInternalStateResponse {
-  // Internal state of the SLAM algorithm required to continue mapping/localization
+  // A chunk of the internal state of the SLAM algorithm required to continue
+  // mapping/localization
   bytes internal_state = 1;
 }
+
+message GetPointCloudMapStreamRequest {
+  // Name of slam service
+  string name = 1;
+}
+
+message GetPointCloudMapStreamResponse {
+  // pointclouds are returned in chunks as a set of bytes which can be
+  // aggrigated into the standard PCD format
+  // https://pointclouds.org/documentation/tutorials/pcd_file_format.html
+  bytes point_cloud_pcd_chunk = 1;
```

Comment:
Given that each `GetPointCloudMapStreamResponse`  messages will now only contain a chunk of the PCD file, I changed the names of this message member to include `_chunk` in order to prevent a caller from thinking that it contains an entire PCD file.

Reply:
Will `GetPointCloudMapStreamResponse` be the final name or will this be reverted to `GetPointCloudMapResponse` down the line?

Reply:
We will change the name to `GetPointCloudMapResponse`.